### ISSUE TITLE
Possibly confusing wording regarding a separator's orientation in menus and menubars

### ISSUE THREAD

**Issue by charlie-yao** [w3c/aria-practices#2197]
In the authoring practices 1.1 document, there's a bullet point for menus/menubars that says:

> All `separators` should have `aria-orientation` consistent with the separator's orientation.

For me at least, this statement seems rather confusing due to its tautological nature. At first I thought it was a typo, and that it should read something like, "...consistent with the parent menu's orientation", but that also doesn't seem to make sense. If a menu or menubar is oriented vertically, then their separator elements should be oriented horizontally, and vice-versa, right?

At least that's the case in the second menubar example, linked below. The menu elements have a default orientation of vertical, and the separator elements have a default orientation of horizontal.

See:
https://www.w3.org/TR/wai-aria-practices-1.1/#menu
https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-2/menubar-2.html

**Comment by JAWS-test** [comment]
It is not tautological, because it is meant that aria-orientation should be consistent with **visual** orientation

**Comment by charlie-yao** [comment]
Oh, I see. Thanks for the clarification.

I suppose the wording could changed to (hopefully) reduce ambiguity? Maybe something like:

> All `separators` should have an `aria-orientation` that's consistent with its visual orientation.

**Comment by a11ydoer** [comment]
@charlie-yao thanks for your feedback. APG group will look into your suggesiton. 
By the way, [ARIA APG now have 1.2 version](https://www.w3.org/TR/wai-aria-practices-1.2/) which is synchronized with ARIA 1.2 Candidate recommendaion.

**Comment by a11ydoer** [comment]
admin note - add this to menu bar project for follow up.
